src/util/argParser.ts

```diff
@@ -156,6 +160,14 @@ class ArgParser {
             "Allow Hashtag URLs, useful for single-page-application crawling or when different hashtags load dynamic content",
         },
 
+        selectLinks: {
+          describe:
+            "one or more selectors for extracting links, in the format [css selector]->[property to use],[css selector]->@[attribute to use]",
```

Comment:
```suggestion
            "One or more selectors for extracting links, in the format [css selector]->[property to use],[css selector]->@[attribute to use]",
```
We're admittedly not super consistent about this, but most of the descriptions seem to use sentence casing